README.md

```diff
@@ -110,7 +121,7 @@ kotlin {
 To use the library in a JVM-only project add the platform to the artifact name, e.g.:
 
 ```groovy
-implementation("org.jetbrains.kotlinx:kotlinx-benchmark-runtime-jvm:0.2.0-dev-20")
+implementation("org.jetbrains.kotlinx:kotlinx-benchmark-runtime-jvm:0.3.0")
```

Comment:
Thanks to gradle metadata, the root artifact name can be used everywhere.

Reply:
Rephrased source set dependency delaration and removed platform specific suffix note